tests/helpers/mk-tmp-dir-in.js

```diff
@@ -1,14 +0,0 @@
-'use strict';
-
-const fs = require('fs-extra');
-const temp = require('temp');
```

Comment:
Is `temp` still used elsewhere?

Reply:
yes temp is actually a dependency and is used in the `ember test` command (for some reason 🤷)